docs/output.md

```diff
@@ -582,9 +589,9 @@ For all samples:
 **Output directory: `results/Reports/[SAMPLE]/SamToolsStats`**
 
 - `[SAMPLE].bam.samtools.stats.out`
-  - RAW statistics used by MultiQC
+  - Raw statistics used by `MultiQC`
 
-For further reading and documentation see the [samtools manual](https://www.htslib.org/doc/samtools.html#COMMANDS_AND_OPTIONS)
+For further reading and documentation see the [`samtools` manual](https://www.htslib.org/doc/samtools.html#COMMANDS_AND_OPTIONS)
 
 #### bcftools stats
 
```

Comment:
[bcftools](https://samtools.github.io/bcftools/) is a program for variant calling and manipulating files in the `Variant Call Format`.
Plot will show:

or change `Variant Call Format` to `VCF`